e2e_testing/torchscript/elementwise.py

```diff
@@ -479,3 +479,39 @@ def forward(self, a):
 @register_test_case(module_factory=lambda: ElementwiseRsqrtModule())
 def ElementwiseRsqrtModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(3, 4))
+
+# ==============================================================================
+
+class ElementwiseAbsModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+    @export
+    @annotate_args([
+        None,
+        ([-1], torch.float32, True),
+    ])
+
+    def forward(self, a):
+        return torch.abs(a)
+
+@register_test_case(module_factory=lambda: ElementwiseAbsModule())
+def ElementwiseAbsModule_basic(module, tu: TestUtils):
+    module.forward(torch.tensor([-1, -2, -3], dtype=torch.float32))
```

Comment:
nit: prefer to use `tu.rand` to generate random tensors for testing when possible. Probably make the tensor larger like tu.rand(3,4,5) so that it's more likely both negative and possible numbers are generated.

Reply:
Note: `tu.rand` defaults to the range 0 to 1. To get negative values you must specify it as an argument 

https://github.com/llvm/torch-mlir/blob/56c6e3676bbe1571dab0c7cc1fb5080846819438/python/torch_mlir_e2e_test/torchscript/framework.py#L151

Reply:
Thanks for the comment!